docs/upgrade/xenial_prep.rst

```diff
@@ -0,0 +1,193 @@
+
+Upgrading from version 0.3.x - reprovisioning your Admin Workstation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your Admin Workstation has not been updated since version 0.3 of SecureDrop was released, its Tails version is most likely also out-of-date. In this case, we recommend provisioning a new Admin Workstation using the configuration information from the old workstation.
+
+First, prepare a new Tails USB stick with a persistent volume, using the latest version of Tails. For more information on this process, see Docs: Creating Tails USBs. This will be your new Admin Workstation.
+
+Start up your new Admin Workstation with its persistent volume unlocked and an administration password set.
+
+Open a terminal and run the following commands to install the SecureDrop app code:
+
+gpg --recv-key "2224 5C81 E3BA EB41 38B3 6061 310F 5612 00F4 AD77"
+cd ~/Persistent
+git clone https://github.com/freedomofpress/securedrop.git
+cd ~Persistent/securedrop
+git checkout 0.11.0
+git tag -v 0.11.0
+
+
+You should see Good signature from "SecureDrop Release Signing Key" in the output of that last command, along with the fingerprint above.
+
+If you do not, signature verification has failed and you should not proceed with the installation. If this happens, please contact us at securedrop@freedom.press.
+
+Next, mount the persistent volume of the old Admin Workstation in order to retrieve instance-specific files that you’ll need to set up the new workstation.
+
+To do so:
+Plug your old Admin Workstation into a free USB port
+Browse to Places > Computer in the Tails top navigation bar
+The old Admin Workstation’s persistent volume will appear in the left-hand menu, listed as an encrypted volume. Click the listing, and enter the decryption passphrase for the volume to mount it.
+
+Next, copy the files that you’ll need for the new Admin Workstation. Open a terminal and run the following commands:
+
+cp /media/amnesia/TailsData/openssh-client/* ~/.ssh/
+export SRC="/media/amnesia/TailsData/install_files/ansible_base"
+export DST="~/Persistent/securedrop/install_files/ansible-base"
+cp $SRC/{app,mon}* $DST/
+cp $SRC/prod-specific.yml $DST/
+
+Next, you’ll need to copy over the instance’s submission key and OSSEC public key. Their filenames may vary, but you can check them in the instance configuration file using the following command:
+
+grep "_public_key" $DST/prod-specific.yml
+
+Assuming that their names are Securedrop.asc and ossec.asc respectively, you should then copy them across by running the following commands:
+
+cp $SRC/Securedrop.asc $DST/
+cp $SRC/ossec.asc $DST/
+
+If you use Tails’ KeepassX password manager to store instance-specific passwords, you should also copy over the old workstation’s KeepassX database. 
+The default location for the KeepassX database is /media/amnesia/TailsData/Persistent/securedrop-keepassx.kdbx. Copy it to the new Admin Workstation’s persistent volume with the following command:
```

Comment:
My concern is that it may not have been saved with that name or in that exact location, giving the default location encourages  people to poke about and find it if they saved it elsewhere.

Reply:
Fair. We should then add a comment like that saying "you may have placed it elsewhere so you'll need to change the next command blah blah"